### PR TITLE
Chat2Query rename to SQL Editor

### PR DESCRIPTION
**Pull request by shizn**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Rename Chat2Query into SQL Editor

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 1]
```diff
@@ -26,27 +26,25 @@ Additionally, you can try out TiDB features on [TiDB Playground](https://play.ti
...
 
 2. To try the AI capacity of TiDB Cloud, follow the on-screen instructions to allow PingCAP and OpenAI to use your code snippets for research and service improvement, and then click **Save and Get Started**.
 
-3. In the editor, you can either simply type `--` followed by your instructions to let AI automatically generate SQL queries, or write SQL queries manually.
+3. In SQL Editor, press **⌘ + I** on macOS (or **Control + I** on Windows or Linux) to instruct [Chat2Query (beta)](tidb-cloud/tidb-cloud-glossary.md#chat2query) to generate SQL queries automatically.
 
-    For example, to create a new table `test.t` with two columns (column `id` and column `name`), you can type `use test;` to specify the database, type `-- create a new table t with id and name` as the instruction, and then press **Enter** to let AI generate a SQL statement accordingly. 
+    For example, to create a new table `test.t` with two columns (column `id` and column `name`), you can type `use test;` to specify the database, type `create a new table t with id and name` as the instruction, and then press **Enter** to let AI generate a SQL statement accordingly.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-quickstart.md:45]: > type `create a new table t with id and name` as the instruction, and then press **Enter** to let AI generate a SQL statement accordingly.

This needs to be modified according to the console behavior.

File: tidb-cloud/explore-data-with-chat2query.md [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
 ---
 
-# Explore Your Data with AI-Powered Chat2Query (beta)
+# Explore Your Data with AI-Assisted SQL Editor
 
-TiDB Cloud is powered by AI. You can use Chat2Query (beta), an AI-powered SQL editor in the [TiDB Cloud console](https://tidbcloud.com/), to maximize your data value.
+You can use the built-in AI-assisted SQL Editor in the [TiDB Cloud console](https://tidbcloud.com/) to maximize your data value.
 
-In Chat2Query, you can either simply type `--` followed by your instructions to let AI generate SQL queries automatically or write SQL queries manually, and then run SQL queries against databases without a terminal. You can find the query results in tables intuitively and check the query logs easily.
+In SQL Editor, you can either write SQL queries manually or simply press **⌘ + I** on macOS (or **Control + I** on Windows or Linux) to instruct [Chat2Query (beta)](/tidb-cloud/tidb-cloud-glossary.md#chat2query) to generate SQL queries automatically. This enables you to run SQL queries against databases without a local SQL client. You can intuitively view the query results in tables or charts and easily check the query logs.
```
Comment (lilin90) [inline tidb-cloud/explore-data-with-chat2query.md:10]:
```suggestion
In SQL Editor, you can either write SQL queries manually or simply press <kbd>⌘</kbd> + <kbd>I</kbd> on macOS (or <kbd>Control</kbd> + <kbd>I</kbd> on Windows or Linux) to instruct [Chat2Query (beta)](/tidb-cloud/tidb-cloud-glossary.md#chat2query) to generate SQL queries automatically. This enables you to run SQL queries against databases without a local SQL client. You can intuitively view the query results in tables or charts and easily check the query logs.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 3]
```diff
@@ -26,27 +26,25 @@ Additionally, you can try out TiDB features on [TiDB Playground](https://play.ti
...
 
-In Chat2Query, you can either simply type `--` followed by your instructions to let AI automatically generate SQL queries, or write SQL queries manually and run them against databases without using a terminal.
-
-1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click on a cluster name to go to its overview page, and then click **Chat2Query** in the left navigation pane.
+1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click on a cluster name to go to its overview page, and then click **SQL Editor** in the left navigation pane.
 
 2. To try the AI capacity of TiDB Cloud, follow the on-screen instructions to allow PingCAP and OpenAI to use your code snippets for research and service improvement, and then click **Save and Get Started**.
 
-3. In the editor, you can either simply type `--` followed by your instructions to let AI automatically generate SQL queries, or write SQL queries manually.
+3. In SQL Editor, press **⌘ + I** on macOS (or **Control + I** on Windows or Linux) to instruct [Chat2Query (beta)](/tidb-cloud/tidb-cloud-glossary.md#chat2query) to generate SQL queries automatically.
```
Comment (lilin90) [inline tidb-cloud/tidb-cloud-quickstart.md:43]:
```suggestion
3. In SQL Editor, press <kbd>⌘</kbd> + <kbd>I</kbd> on macOS (or <kbd>Control</kbd> + <kbd>I</kbd> on Windows or Linux) to instruct [Chat2Query (beta)](/tidb-cloud/tidb-cloud-glossary.md#chat2query) to generate SQL queries automatically.
```

File: tidb-cloud/explore-data-with-chat2query.md [hunk 4]
```diff
@@ -101,9 +121,32 @@ After running the queries, you can see the query logs and results immediately at
...
+## Rewrite SQL queries using Chat2Query
+
+In SQL Editor, you can use Chat2Query to rewrite existing SQL queries to optimize performance, fix errors, or meet other specific requirements.
+
+1. Select the lines of SQL queries you want to rewrite with your cursor.
+
+2. Invoke Chat2Query for rewriting by using the keyboard shortcut for your operating system:
+
+    - **⌘ + I** on macOS
+    - **Ctrl + I** on Windows or Linux
```
Comment (lilin90) [inline tidb-cloud/explore-data-with-chat2query.md:133]:
```suggestion
    - <kbd>⌘</kbd> + <kbd>I</kbd> on macOS
    - <kbd>Control</kbd> + <kbd>I</kbd> on Windows or Linux
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by Oreoxmt — APPROVED** [review]

File: tidb-cloud/explore-data-with-chat2query.md [hunk 5]
```diff
@@ -101,9 +121,32 @@ After running the queries, you can see the query logs and results immediately at
...
+In SQL Editor, you can use Chat2Query to rewrite existing SQL queries to optimize performance, fix errors, or meet other specific requirements.
+
+1. Select the lines of SQL queries you want to rewrite with your cursor.
+
+2. Invoke Chat2Query for rewriting by using the keyboard shortcut for your operating system:
+
+    - <kbd>⌘</kbd> + <kbd>I</kbd> on macOS
+    - <kbd>Control</kbd> + <kbd>I</kbd> on Windows or Linux
+
+  Press **Enter** after providing your instructions to let AI handle the rewrite.
```
Comment (Oreoxmt) [inline tidb-cloud/explore-data-with-chat2query.md:135]:
```suggestion
    Press **Enter** after providing your instructions to let AI handle the rewrite.
```

**Pull request merged**